ui/src/hooks/useDynamicScript.js

```diff
@@ -40,4 +40,21 @@ const useDynamicScript = (args) => {
   };
 };
 
+// Renderless component wrapper
+export const LoadDynamicScript = (props) => {
+  const { ready, failed } = useDynamicScript({
+    url: props.url,
+  });
+
+  useEffect(() => {
+    if (props.url) {
+      props.setConfigStatusReady(ready);
```

Comment:
`setConfig...` is a bit strange for this shared component. Can we explicitly capture the arguments in the function signature?

```javascript
export const LoadDynamicScript = (url, setConfigReady, setStatusFailed) => {
}
```